core/src/main/groovy/com/craigburke/document/core/Document.groovy

```diff
@@ -41,4 +45,44 @@ class Document extends BlockNode {
     List children = []
     List<EmbeddedFont> embeddedFonts = []
 
+    /**
+     * Set width and height of the document.
+     *
+     * @param arg a Dimension instance, a List<Number> [width, height] or the name of a standard paper size ("a4", "letter", "legal")
+     */
+    void setSize(def arg) {
```

Comment:
This is great!

Can you change this to 3 individually overloaded setSize methods? `setSize(Dimension size)` `setSize(List size)` etc?